app/helpers/products_helper.rb

```diff
@@ -1,4 +1,11 @@
 # frozen_string_literal: true
 
 module ProductsHelper
+  def total_quantity_of_products(product)
+    total_quantity = []
+    product.stocks.each do |stock|
+      total_quantity << stock.quantity
+    end
+    total_quantity.sum
+  end
```

Comment:
Helpers should be used when some frontend-related logic is too complex to put directly in a view. 

Calculating `total_quantity` is not really a frontend logic. Wouldn't it be nicer to be able to call it like `product.total_quantity`? Let's move this logic to the model.

Also I think you can refactor this code to use `map` :)